External/FEXCore/Source/Interface/Core/Interpreter/InterpreterCore.cpp

```diff
@@ -26,19 +34,30 @@ namespace FEXCore::IR {
 namespace FEXCore::CPU {
 class CPUBackend;
 
-static void InterpreterExecution(FEXCore::Core::CpuStateFrame *Frame) {
-  auto Thread = Frame->Thread;
+InterpreterCore::InterpreterCore(FEXCore::Context::Context *ctx, FEXCore::Core::InternalThreadState *Thread)
+  : CPUBackend(Thread, 1024 * 1024 * 16, 1024 * 1024 * 128)
```

Comment:
The Min and Max sizes here should be using constant defines to make it clearer, like it was before.
Same for the x86 and AArch64 sides.

Reply:
I'll move them out

Reply:
Huh, I actually left the consts there, but didn't use them. Good catch!

Moved them to be local to JIT.cpps, changed code to use them.